src/app/(Authentication)/Login/Email/styles.ts

```diff
@@ -0,0 +1,75 @@
+import { StyleSheet } from 'react-native';
+
+export default StyleSheet.create({
+  container: {
+    backgroundColor: '#fff',
+    width: '100%',
+    height: '100%',
+    alignItems: 'center',
+    justifyContent: 'flex-start',
+  },
+  backText: {
+    color: '#000',
+    fontSize: 16,
+    opacity: 0.3,
+    fontWeight: '400',
+  },
+  backButton: {
+    marginTop: 100,
+    marginBottom: 40,
+    marginRight: 280,
+    padding: 10,
+  },
+  instructionText: {
+    color: '#000',
+    fontSize: 24,
+    fontWeight: '800',
+    marginRight: 18,
+    marginLeft: 18,
+  },
+  emailText: {
+    marginTop: 90,
+    marginRight: 200,
+    fontSize: 13,
+    opacity: 0.35,
+  },
+  input: {
+    marginTop: 5,
+    height: 40,
+    borderBottomWidth: 2,
+    padding: 10,
+    width: 310,
+    borderRadius: 5,
+    borderColor: '#920E1A',
```

Comment:
`borderColor` should depend on whether the user is typing or not! It should be red if typing and black otherwise